dom0/sd-log-disable-plugin.sls

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# vim: set syntax=yaml ts=2 sw=2 sts=2 et :
+
+##
+# Disable securedrop-log rsyslog plugin in sd-log vm
```

Comment:
Could we also link to the issue in the comments here? It would be useful to note, for future maintainers, that this is the side effect of using the same package for both the client and server component of the log: we want to make sure that sd-log does not send logs.